### ISSUE TITLE
Options for getSimilarWords and getNeighbors are not taken into account

### ISSUE THREAD

**Issue by FerdinandPiette** [LeeXun/word2vector#3]
Hi,

I tried this code : 
```
const word2vec = require('word2vector');
const model = 'data/frWac_postag_no_phrase_700_skip_cut50.bin';
const word = 'yoga_n';

word2vec.load(model);
console.log(word2vec.getSimilarWords(word, {N:5}).length);
console.log(word2vec.getNeighbors(word2vec.getVector(word), {N:5}).length);
```

It displays 40 and 40. 
In src/w2vLeeXun.cc, the number of words to be shown is set to 40 and never change.
`const long long N = 40; // number of closest words that will be shown`

**Comment by LeeXun** [comment]
Thanks for reporting. Will check asap.

**Comment by nowherenone** [comment]
# #4  

@LeeXun - I made a fix for this issue, please take a minute to review it and merge.  Thank you! :)

PS: Also I've changed the behavior when getSimilarWords is receiving an unknown word: before it was exiting the NodeJS process, but now it just returns nothing -  I believe that makes sense.

**Comment by LeeXun** [comment]
Special thanks to @nowherenone 

Merged in v2.1.0!
